doc/man/remotectl.xml

```diff
@@ -44,12 +44,12 @@
   </refsynopsisdiv>
 
 
-  <refsect1>
+  <refsect1 id="remotectl-description">
```

Comment:
I'm confused -- the commit message says "remove generated IDs" but this commit *adds* IDs?